### ISSUE TITLE
Pass config to sandboxed worker

### ISSUE THREAD

**Issue by PaulGrimshaw** [taskforcesh/bullmq#195]
Hi

Is there a way to pass some configuration to a sandboxed worker, that can be accessed inside the worker?  Or alternatively can you retrieve the queue name from inside the worker?  I use the same worker definition for a few different queues, but need to pick up some settings to indicate how it should run.

If the worker is not sandboxed, you can retrieve the queue name via `job.queue.name` but as soon as this worker is sandboxed the job no longer has the `queue` property.

Basically i'm struggling to see how any data can be passed or retrieved in the processor when sandboxed - bullmq takes the job, fires up the processor and sends the job in. Seems a bit overkill to always have to place the `queue-name` into the job payload too?  

What am i missing?

**Comment by PaulGrimshaw** [comment]
Ok i have worked around this for now - added a prefix to the job id for the queue name.  Bit messy but will do for now
